README.md

```diff
@@ -8,9 +8,7 @@ If you've come here because you want _Swift_ or _Python_ classes for FHIR data m
 - [Swift-FHIR][] and [Swift-SMART][]
 - Python [client-py][]
 
-The `master` branch is currently capable of parsing _STU 3, v3.0.0_.  
-The `develop` branch should be capable of parsing the continuous integration build and will be merged into master on new major FHIR releases.
-There may be tags for specific freezes, see [releases](https://github.com/smart-on-fhir/fhir-parser/releases).
+The `master` branch is currently capable of parsing _R5_.
```

Comment:
The `develop` branch is strictly 14 commits behind `master` currently - folks seem to have switched to trunk-based dev (which seems fine, since it's currently backwards compatible).

I took out the link to releases because (a) there are no release, just tags and (b) even the tags aren't very up to date - just a bunch of 1.x stuff from early days.